content/articles/run-length-encoding/index.md

```diff
@@ -0,0 +1,150 @@
+### Introduction
+
+In this article, we will look at how Run Length Encoding works in helping us reduce our file sizes.
+Before delving into the topic, we will first get to know what is compression.
+
+**Compression** - This is a process where a file size is reduced using algorithms resulting in a file that uses fewer storage bits than the original file.
+For example, in a situation where one wants to send a picture to his/her friend, compression will be done at the source device and decompressed at the destination device. This enables the file to be sent faster and additionally, it reduces overhead traffic for transmitting the data.
+
+Another example of compression is in social media apps. It is applied in applications such as WhatsApp where one will notice the image he/she sent is of lower quality and consumes much lesser space. Other apps also give users options in data saving settings for whether choosing whether the images and videos loaded should be of low quality or high quality. 
+
+We also notice compression in video streaming sites where videos are loaded in low quality when there is low internet connectivity and vice versa. 
+
+All of these are some of the few applications of data compression.
+
+#### Types of Compression algorithms.
+
+There two main classification types for compression algorithms are:
+
+* **Lossy**
+* **Lossless**
+
+We use lossless algorithms where we want to retain our information and the quality.
+The processes done by these algorithms are reversible and of very low compression ratios because we don't lose any information.
```

Comment:
Don't write every sentence in a new line.

Combine sentences talking about one idea into a paragraph

Reply:
Do this for all sentences which you see fit